### PR TITLE
Orders and Shipments Articles Review by AW

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/commerce/2.x/en/orders-and-fulfillment/orders/order-management-statuses-reference-guide.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Order Management Statuses Reference Guide
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/orders/order-management-statuses-reference-guide.md:1]: @andrewwilkinsonLR @justinchoi001 - it _looks_ like this article was renamed from `order management statuses` -> `order management statuses reference guide`. What's the rationale for changing the name? (assuming the content is all the same?)

File: docs/commerce/2.x/en/orders-and-fulfillment/shipments/creating-a-shipment.md [hunk 2]
```diff
@@ -91,5 +91,11 @@ The order manager can update the status of the shipment as needed, until the ord
...
 * [Setting Inventory by Warehouse](../../managing-a-catalog/managing-inventory/setting-inventory-by-warehouse.md)
-* [Order Life Cycle](../orders/order-life-cycle.md)
+* [Introduction to Shipments](./introduction-to-shipments.md)
+* [Cancelling a Shipment](./cancelling-a-shipment.md)
+* [Processing an Order](../orders/processing-an-order.md)
 * [Order Information](../orders/order-information.md)
+* [Order Life Cycle](../orders/order-life-cycle.md)
+* [Orders Menu Reference Guide](../orders/orders-menu-reference-guide.md)
+* [Products Overview](../../managing-a-catalog/creating-and-managing-products/products/products-overview.md)
+* [Warehouse Reference Guide](../../managing-a-catalog/managing-inventory/warehouse-reference-guide.md)
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/shipments/creating-a-shipment.md:101]: Please reconsider in the future adding so many links in this "Additional Information" section. No more than 5 in the future. Really feels like we're throwing in the kitchen sink.

File: docs/commerce/2.x/en/orders-and-fulfillment/shipments/introduction-to-shipments.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Introduction to Shipments
 
-Create shipments first in the _Orders_ menu as part of processing an order. Once an order is processed, you are redirected to the _Shipments_ menu to complete order processing.
+Create shipments first in the _Orders_ menu as part of [processing an order](../orders/processing-an-order.md). Once an order is processed, you are redirected to the _Shipments_ menu to complete oder processing.
```
Comment (jrhoun) [inline docs/commerce/2.x/en/orders-and-fulfillment/shipments/introduction-to-shipments.md:3]: One typo introduced: `oder`

**Comment by jrhoun** [comment]
Looks like there are unrelated files that have been added to this pull. Therefore, closing this out and asking that the feedback be addressed and be re-sent.

**Comment by andrewwilkinsonLR** [comment]
Okay. Will take a look.

**Pull request closed without merging**